app/Models/Actor.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Actor extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'name',
+        'surname',
+        'patronymic',
+        'date_of_birth',
+        'growth',
+    ];
+
+    protected $dates = [
+        'created_at',
+        'updated_at',
```

Comment:
чтобы дата из базы данных, сразу преобразовывалась в объект даты
```suggestion
        'created_at',
        'updated_at',
        'date_of_birth',
```